### ISSUE TITLE
Survey Publish/close buttons

### ISSUE THREAD

**Issue by rebecarubio** [zetkin/app.zetkin.org#984]
## Description
A user should be able to easily publish and unpublish a survey clicking a button. 

## Relevant Job Stories
* When a survey is no longer relevant, I want to close the survey, so that no more people can respond to it
* When I’m designing my survey, I want to make sure it’s not visible to respondents, so that I can make changes without risking affecting responses

## Prerequisites
Don't start until #982 is done

## Requirements
- [x] Add "publish survey" button in the header a survey 
- [x] If survey doesn't have any question, "publish survey" button it's disabled 
- [x] When survey is published the button text should change to "unpublish survey"

## Design specifications
[Figma](https://www.figma.com/file/EjLcpjKWgNizCb5PRBF51P/Zetkin-3?node-id=2517%3A36483&t=Zm38Y7wFONKOsZfv-0)

### Button disabled (no questions in survey)
![disabled survey](https://user-images.githubusercontent.com/36491300/217783365-f0118c4f-6f23-4841-8b84-2d876b6f0547.png)

### Button active
![active button](https://user-images.githubusercontent.com/36491300/217783589-d5998d8d-4460-4d59-8310-84f4005f6ad1.png)

### Button to unpublish survey
![unpublish](https://user-images.githubusercontent.com/36491300/217784179-37fe6933-64d6-4173-93da-a340df619600.png)

## Open questions
* What's the best terminology for this, i.e. the best label for the open and close buttons respectively?
